source/_components/ads.markdown

```diff
@@ -0,0 +1,62 @@
+---
+layout: page
+title: "ADS"
+description: "Connect Home Assistant to TwinCAT devices via the ADS interface."
+date: 2016-05-25 23:39
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
+ha_category: Hub
+ha_release: 0.55
+ha_iot_class: "Local Push"
+---
+
+The ADS (automation device specification) describes a device-independent and
+fieldbus-independent interface for communication between Beckhoff® automation
+devices running TwinCAT® and other devices implementing this interface.
+
+Configuration parameters:
+
+- **device** (*Required*): The AMS NetId that identifies the device.
```

Comment:
Please use the configuration tags for listing configuration variables. For more information:
https://home-assistant.io/developers/documentation/create_page/#configuration